test/ingestors/highlight.test.ts

```diff
@@ -0,0 +1,119 @@
+import { expect } from 'chai';
+import { HighlightIngestor } from '../../src/ingestors/highlight';
+import { mintIngestorResources } from '../../src/lib/resources';
+import { EVMMintInstructions } from '../../src/lib/types/mint-template';
+import { MintTemplateBuilder } from '../../src/lib/builder/mint-template-builder';
+import { basicIngestorTests } from '../shared/basic-ingestor-tests';
+
+const resources = mintIngestorResources();
+
+describe('highlight', function () {
+  basicIngestorTests(new HighlightIngestor(), resources, {
+    successUrls: [
+      'https://highlight.xyz/mint/665fa33f07b3436991e55632',
+      'https://highlight.xyz/mint/66856628ff8a01fdccc132f4',
+    ],
+    failureUrls: [
+      'https://highlight.xyz/mint/66963c500b48236f1acf322b',
+      'https://foundation.app/mint/base/the-billows',
+    ],
+    successContracts: [
+      { chainId: 8453, contractAddress: '0x0E5DDe3De7cf2761d8a81Ee68F48410425e2dBbA' },
+      { chainId: 8453, contractAddress: '0xBE96B2572CA0F1ac8ec6323Bc9037AffD270bA7F' },
+    ],
+    failureContracts: [{ chainId: 5000, contractAddress: '0x62F8C536De24ED32611f128f64F6eAbd9b82176c' }],
```

Comment:
Recommend adding the final parameter here to ensure these tests always pass, declaring the blockhash to run simulation as of (this is the current blockhash atow)

```
{
   8453: '0x108D2CF'
}
```